config/jobs-chromeos.yaml

```diff
@@ -379,7 +379,7 @@ _anchors:
     params: &watchdog-reset-job-params
       bl_message: 'coreboot-'
       wdt_dev: 'watchdog0'
-    kcidb_test_suite: kernelci_watchdog_reset
+    # kcidb_test_suite: kernelci_watchdog_reset
```

Comment:
Should this property be dropped entirely from the config or will it be still necessary in the future?

Reply:
It's just a testing commit which will be discarded later. Please check the commit message.

Reply:
Deleted the commit as verification is completed.